vectordb_bench/frontend/components/run_test/caseSelector.py

```diff
@@ -103,10 +103,12 @@ def caseConfigSetting(st, dbToCaseClusterConfigs, uiCaseItem: UICaseItem, active
                 elif config.inputType == InputType.Float:
                     caseConfig[config.label] = column.number_input(
                         config.displayLabel if config.displayLabel else config.label.value,
-                        step=config.inputConfig.get("step", 0.1),
-                        min_value=config.inputConfig["min"],
-                        max_value=config.inputConfig["max"],
-                        key=key,
+                        value=config.inputConfig["value"],
+                        help=config.inputHelp,
+                    )
```

Comment:
Could you please share if there is any specific reason for considering the removal of "min" / "max" / "step" / "key"?

Reply:
Thanks for pointing out, I made a mistake here while adding a new input type. I've reverted these changes. Can you please review it again.